libs/shared/adapters/src/lib/algorand-adapter.ts

```diff
@@ -215,7 +217,10 @@ export class AlgorandAdapter {
     const confirmedRound: number = info['confirmed-round'] || 0
     const poolError: string = info['pool-error'] || ''
     const assetIndex: number = info['asset-index'] || 0
-    return { confirmedRound, poolError, assetIndex }
+    const applicationIndex: number =
+      info['application-index'] || info['txn']?.['txn']?.['apid'] || 0
+
+    return { confirmedRound, poolError, assetIndex, applicationIndex }
```

Comment:
For create app transactions, the `application-index` prop is set. For no-op call app transactions, we need to dig into the transaction params a bit. Need it for both (create+setup) cases to ensure the status is updated accordingly for both cases (and potentially other cases later on).